src/scss/grommet-core/_objects.form.scss

```diff
@@ -73,4 +73,21 @@
 
 .#{$grommet-namespace}form--compact {
   max-width: $compact-form-width;
+
+  // Fixes a bug in Windows which causes overflow.
+  label.#{$grommet-namespace}check-box {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+  }
+
+  label.#{$grommet-namespace}radio-button {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+
+    .#{$grommet-namespace}radio-button__control {
+      min-width: 24px;
```

Comment:
can you expand on why this is needed? Also I believe we should be using the inuit base spacing unit.

Reply:
It would be great if we can find a solution that does not have a link between form -> check-box and radio-button.

Reply:
The issue is specific to compact form + check/radio box. It would be reasonable to only target the elements within compact form as that would disrupt the least amount of existing  code bases.